package.json

```diff
@@ -0,0 +1,21 @@
+{
+  "name": "@codexteam/surveys",
+  "version": "0.0.0",
+  "type": "module",
+  "scripts": {
+    "dev": "vite",
+    "build": "tsc && vite build",
+    "preview": "vite preview",
+    "prettier": "prettier . --write",
+    "lint": "yarn run eslint src --fix"
+  },
+  "devDependencies": {
+    "eslint": "^8.27.0",
+    "eslint-config-codex": "^1.7.0",
+    "prettier": "2.7.1",
+    "typescript": "*",
```

Comment:
Could you add some TS version here? For example:
```suggestion
    "typescript": "^4.9.3",
```
You can do it with `yarn add -D typescript@^4.9.3` command.

Reply:
As you can see in yarn.lock file, the version 4.8 is used in the project, but it will be better to have this version in package.json file.
<img width="354" alt="image" src="https://user-images.githubusercontent.com/37909603/204275279-a77af001-80da-4f67-97b1-a3749f36176d.png">

Reply:
I'm not sure, that everything will work fine with 4.9, so you can add ^4.8.

Reply:
Added 4.9.3, It's worked too